frontend/package.json

```diff
@@ -16,19 +16,21 @@
     "grunt-contrib-watch": "^1.0.0",
     "grunt-filerev": "^2.3.1",
     "grunt-include-source": "^1.0.0",
+    "grunt-ng-annotate": "^3.0.0",
     "grunt-usemin": "^3.1.1",
     "grunt-wiredep": "^3.0.1",
     "http-server": "^0.9.0",
```

Comment:
Let's check whether we need http-server included here, as far as I can see you've added spa-http-server, is this one just an addition to the regular http-server package or independent one?